ABWL-LEDER/appendix/LKF.md

```diff
@@ -0,0 +1,440 @@
+# Lernkontrollfragen Allgemeine Betriebswirtschaftslehre
+
+## Skript 1
+
+### Produktionsfaktoren, betriebliche Funktionsbereiche
+
+#### Sie kennen die Instrumente der Erstellung einer Ablauforganisation
+
+- **Kommentar: wahrscheinlich ist Aufbauorganisation gemeint?**
+
+#### Sie können Aufgabenanalyse und –Synthese zum Erstellen der Inhalte einer Ablauforganisation erläutern
+
+- **Kommentar: ebenso?**
```

Comment:
```suggestion
#### Sie kennen die Instrumente der Erstellung einer Aufbauorganisation

- Stellenbeschreibung
- Organigramm
- Informationsfluss
- Umsetzung von Verantwortung

#### Sie können Aufgabenanalyse und –Synthese zum Erstellen der Inhalte einer Aufbauorganisation erläutern

**Aufgabenanalyse**

- Ziele und dazu gehörenden Aufgaben des Unternehmens analysieren
- Unterteilung der Aufgaben in Teilaufgaben und die jeweils erforderlichen Arbeitsschritte

**Aufgabensynthese**

- Teilaufgaben zu sinnvollen Aufgabenkomplexen zusammenfassen
- definieren, welche Stelle, welchen Aufgabenkomplex übernehmen soll
```

Reply:
Ich denke auch, dass hier in beiden Fällen die Aufbauorganisation gemeint ist. Arbeitsanalyse und -synthese sind Teil der Ablauforganisation.

![Bildschirmfoto 2021-11-08 um 16 21 47](https://user-images.githubusercontent.com/21085384/140769049-8c77fc25-f412-4863-ac41-3d8b5d9c166e.png)